common/pinot/pinotQueryValidator_test.go

```diff
@@ -115,7 +115,7 @@ func TestValidateQuery(t *testing.T) {
 		},
 		"Case8-4: query with custom keyword field not equal": {
 			query:     "CustomKeywordField != 0",
-			validated: "JSON_MATCH(Attr, '\"$.CustomKeywordField\"!=''0''') and JSON_MATCH(Attr, '\"$.CustomKeywordField[*]\"!=''0''')",
+			validated: "(JSON_MATCH(Attr, '\"$.CustomKeywordField\"!=''0''') and JSON_MATCH(Attr, '\"$.CustomKeywordField[*]\"!=''0'''))",
```

Comment:
nit consider using backticks to make things more readable when fighting quotes in quites 

ie 
```
`(JSON_MATCH(Attr, '"$.CustomKeywordField"!=''0''') and JSON_MATCH(Attr, '"$.CustomKeywordField[*]"!=''0'''))`
```

I guess, or similar, I'm having slight trouble following this because I've not spent enough time close to the Pinot component